libkbfs/md_ops.go

```diff
@@ -46,12 +46,19 @@ func (md *MDOpsStandard) convertVerifyingKeyError(ctx context.Context,
 }
 
 func (md *MDOpsStandard) verifyWriterKey(
-	ctx context.Context, rmds *RootMetadataSigned) error {
+	ctx context.Context, rmds *RootMetadataSigned, isTlfFinal bool) error {
 	if !rmds.MD.IsWriterMetadataCopiedSet() {
-		err := md.config.KBPKI().HasVerifyingKey(ctx,
-			rmds.MD.LastModifyingWriter,
-			rmds.MD.WriterMetadataSigInfo.VerifyingKey,
-			rmds.untrustedServerTimestamp)
+		var err error
+		if isTlfFinal {
+			err = md.config.KBPKI().HasUnverifiedVerifyingKey(ctx,
```

Comment:
It's possible the key is verified. But I'm not sure if it matters. I guess it depends on how we handle the UX? We could only alert someone if the client detects it had to verify metadata with an unverified key. But if the UX is, "Warning: this folder is a folder where we can't completely verify all of the metadata, use at your own risk." then it doesn't seem worthwhile.